fvm/environment/accounts_test.go

```diff
@@ -144,29 +144,6 @@ func TestAccounts_GetPublicKeys(t *testing.T) {
 	})
 }
 
-// Some old account could be created without key count register
```

Comment:
@janezpodhostnik could you double check if this is still needed?  we should migrate old accounts / populate the public_key_count field instead of preserving this behaviour.

Reply:
The key count register no longer exists it has been merged into the account status register https://github.com/onflow/flow-go/pull/2799. So this is no longer needed.